### PR TITLE
Feature/dashboard functionality

### PR DESCRIPTION
**Pull request by Humnaa**
Trello Link : https://trello.com/b/FTaaANh4/be-prepared-application

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "node-sass": "^4.12.0",
     "popper.js": "^1.15.0",
     "react": "^16.8.6",
+    "react-cookie": "^4.0.0",
```
Comment (ayousaf11) [inline package.json]: @Humnaa 
is it being used? if not can you please remove it.

File: src/containers/Dashboard/Courses.js [hunk 2]
```diff
@@ -1,32 +1,38 @@
...
-            </Fragment>
-        )          
-    }
+    this.state = {
+      courses: ["OOP", "Algo", "DB"]
+    };
+  }
+  
+  render() {
+    // debugger;
```
Comment (ayousaf11) [inline src/containers/Dashboard/Courses.js]: @Humnaa 
remove comment

File: src/containers/Dashboard/Courses.js [hunk 3]
```diff
@@ -1,32 +1,38 @@
...
+    return (
+      <Fragment>
+        <div>
+          <h5 className="headings ml-5 mb-5 mt-5">Explore BePrepared Skills</h5>
+        </div>
+        <div className="row">
+          <div className="card-container ">
```
Comment (ayousaf11) [inline src/containers/Dashboard/Courses.js]: @Humnaa 
remove extra space at end

File: src/containers/Dashboard/Courses.js [hunk 4]
```diff
@@ -1,32 +1,38 @@
...
+            <div className="courses shadow-lg ml-5 mr-4 mb-5">
+              <Card data={this.state.courses} title="POBLEM SOLVING" />
+            </div>
+            <div className="languages shadow-lg ml-5 mr-2 mb-5">
+              <Card title="LANGUAGES" />
+            </div>
+            <div className="specialized-skills shadow-lg ml-5 mr-5 mb-5">
+              <Card title="SPECIALIZED SKILLS " />
```
Comment (ayousaf11) [inline src/containers/Dashboard/Courses.js]: @Humnaa 
remove extra space at end

File: src/containers/Dashboard/Navbar.js [hunk 5]
```diff
@@ -1,13 +1,17 @@
 import React, { Component } from "react";
 import { Link } from "react-router-dom";
-import { HOME } from "../../constants";
+import { DASHBOARD } from "../../constants";
 import "./Navbar.scss";
 class Navbar extends Component {
+  logout = () => {
+    localStorage.clear();
```
Comment (ayousaf11) [inline src/containers/Dashboard/Navbar.js]: @Humnaa 
Create separate service for Session and move login/logout implementation there.

File: src/containers/Dashboard/Dashboard.js [hunk 6]
```diff
@@ -3,11 +3,14 @@ import Navbar from "./Navbar";
 import ContentContainer from "../Dashboard/ContentContainer";
 
 class Dashboard extends Component {
+
   render() {
+    const email = localStorage.getItem("email");
```
Comment (ayousaf11) [inline src/containers/Dashboard/Dashboard.js]: @Humnaa 
Create separate service for Session and get email from there

File: src/containers/Login/Login.js [hunk 7]
```diff
@@ -48,14 +52,14 @@ class Login extends Component {
       };
       API.userLogin(data, result => {
         if (result.status === "200") {
+          localStorage.setItem("id", result.data.user.uuid);
```
Comment (ayousaf11) [inline src/containers/Login/Login.js]: @Humnaa 

move these in session service

File: src/containers/Dashboard/Dashboard.js [hunk 8]
```diff
@@ -3,11 +3,14 @@ import Navbar from "./Navbar";
...
+    var name = email.substring(0, email.lastIndexOf("@"));
```
Comment (ayousaf11) [inline src/containers/Dashboard/Dashboard.js]: @Humnaa 
Show full name instead

File: package.json (hunk 1 above)
Comment (Humnaa) [reply package.json]: I have removed cookie library. because i'm not using it

File: src/containers/Dashboard/Courses.js (hunk 2 above)
Comment (Humnaa) [reply src/containers/Dashboard/Courses.js]: removed extra comments

File: src/containers/Dashboard/Courses.js (hunk 3 above)
Comment (Humnaa) [reply src/containers/Dashboard/Courses.js]: I have removed extra spaces between bootstrap classes from all components

File: src/containers/Dashboard/Courses.js (hunk 4 above)
Comment (Humnaa) [reply src/containers/Dashboard/Courses.js]: removed

File: src/containers/Dashboard/Navbar.js (hunk 5 above)
Comment (Humnaa) [reply src/containers/Dashboard/Navbar.js]: Created separate folder `service` and move session related implementation there

File: src/containers/Dashboard/Dashboard.js (hunk 6 above)
Comment (Humnaa) [reply src/containers/Dashboard/Dashboard.js]: I have created separate folder service and getting session related info form there

File: src/containers/Login/Login.js (hunk 7 above)
Comment (Humnaa) [reply src/containers/Login/Login.js]: i have moved all session related implementation in a separate folder

File: src/containers/Dashboard/Dashboard.js (hunk 8 above)
Comment (Humnaa) [reply src/containers/Dashboard/Dashboard.js]: Showing name instead of email

**Pull request merged**